src/TLuaInterpreter.cpp

```diff
@@ -301,7 +301,7 @@ double TLuaInterpreter::getVerifiedDouble(lua_State* L, const char* functionName
 }
 
 // No documentation available in wiki - internal function
-// Raises a Lua error in case of an API usage mistake
```

Comment:
This comment incorrectly suggested that this function raised the error - it doesn't!